src/parser.y

```diff
@@ -68,7 +68,7 @@ do {							\
 
 %skeleton "lalr1.cc"
 %defines  /* yacc -d */
-%define api.parser.class {Parser}
```

Comment:
I think you check this file in by mistake. This change is introduced by my duct tape patch solution, but should not be committed but rather reverted after building the compiler.